### PR TITLE
revamp the readme, and add LICENSE + CONTRIBUTORS

### PR DESCRIPTION
**Pull request by purbon**
words, that describe what this does.

it also fixes a typo in the rakefile for the rspec task.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,20 +1,28 @@
-# Integration Testing for Logstash
+# Performance Testing for Logstash
 
 ## Installation
 
-You can use this code as gem within your logstash project, to proceed with the installation you can either download the code and build the gem using the next command:
+You can use this code asd gem within your logstash project, to proceed with the installation you can either download the code and build the gem using the next command:
```
Comment (jsvd) [inline README.md]: as a?

File: README.md [hunk 2]
```diff
@@ -1,20 +1,28 @@
...
 
-## Setup
+## Setup and Runtime
+
+The most simple scenario you could find is using the default set of
+test, available in this gem. To do this you can simple run the ```lsperfm```
+from the root of your Logstash installation.
 
-To run the test you need to next data (you can see an example of them at the `examples/` directory):
+If you like to add you own configurations and test suites,  you need to next data (you can see an example of them at the `examples/` directory):
```
Comment (jsvd) [inline README.md]: need to "next" data?

File: README.md [hunk 3]
```diff
@@ -22,15 +30,14 @@ To run the test you need to next data (you can see an example of them at the `ex
 
 ### Configuration
 
-If you add a file names ```.lsit.yml``` into your main logstash directory
-you can have your configuration and input files in non standard
+If you add a file named ```.lsit.yml``` in your main logstash directory you can have your configuration and input files in non standard
```
Comment (jsvd) [inline README.md]: I think you meant to eliminate all references to "lsit"?

File: README.md [hunk 4]
```diff
@@ -39,22 +46,28 @@ default:
...
 - Run `rake bootstrap` to setup the system.
-- Run `lsit-deps` to install the test dependencies
+- Run `lsperfm-deps` to install the test dependencies
 For 1.4:
 - Run `bin/logstash deps` to setup everything.
 
 ## Performance tests
 
-Test can be run in groups of suites.
+The test are run in groups of suites.
```
Comment (jsvd) [inline README.md]: "Tests are run in groups called suites"?

File: README.md [hunk 5]
```diff
@@ -78,3 +91,29 @@ a suite file defines a series of tests to run.
...
+
+Programming is not a required skill. Whatever you've seen about open source and
+maintainers or community members  saying "send patches or die" - you will not
+see that here.
+
+It is more important to me that you are able to contribute.
+
+### Contribution Steps
+
+1. Test your changes! Run the test suite ('make test')
```
Comment (jsvd) [inline README.md]: "make test"? :)

Comment (purbon) [inline README.md]: makes sense, I just copied from LS :-)

File: README.md (hunk 3 above)
Comment (purbon) [inline README.md]: Well, nowadays the file need to be named like this. This change will need some changes in the gem. I think we can do it here.

**Comment by purbon** [comment]
@jsvd PR updated with your changes, let me know if you are good with it. Thanks a lot for your time reviewing.

File: README.md [hunk 6]
```diff
@@ -1,20 +1,28 @@
...
 
-## Setup
+## Setup and Runtime
+
+The most simple scenario you could find is using the default set of
+test, available in this gem. To do this you can simple run the ```lsperfm```
+from the root of your Logstash installation.
 
-To run the test you need to next data (you can see an example of them at the `examples/` directory):
+If you like to add you own configurations and test suites,  you needuthe next data (you can see an example of them at the `examples/` directory):
```
Comment (jsvd) [inline README.md]: you needuthe next data

File: README.md [hunk 7]
```diff
@@ -78,3 +91,29 @@ a suite file defines a series of tests to run.
...
+
+Programming is not a required skill. Whatever you've seen about open source and
+maintainers or community members  saying "send patches or die" - you will not
+see that here.
+
+It is more important to me that you are able to contribute.
+
+### Contribution Steps
+
+1. Test your changes! Write test and run the test suites.t s
```
Comment (jsvd) [inline README.md]: "the test suites.t s"

**Comment by jsvd** [comment]
LGTM

**Comment by purbon** [comment]
@jsvd thanks for your time man!

**Pull request closed without merging**